docs/getting-started/as-a-developer/README.md

```diff
@@ -112,25 +112,22 @@ import '@rei/cedar/dist/cdr-fonts.css';
 import '@rei/cedar/dist/cedar.css';
 ```
 
-##### PostCSS
-Cedar packages include the [unofficial style field](https://jaketrent.com/post/package-json-style-attribute/) supported by `postcss-import`.
-
-_cedar.postcss_
-```css
-import '@rei/cedar/dist/cdr-fonts.css';
-import '@rei/cedar’;
-```
-
 ##### SCSS
 You can also import CSS from the node_modules folder such as SCSS or another pre-processor.
 
 _cedar.scss_
 ```js
-import '~@rei/cedar/dist/cdr-fonts.css';
-import '~@rei/cedar/dist/cedar.css';
+@import '~@rei/cedar/dist/cdr-fonts.css';
```

Comment:
`@import` isn't correct syntax for js imports which is what this was showing. Do we need to show both js and scss imports instead?

Reply:
the code snippet was just mislabeled as JS, i updated it